cluster/centos/node/scripts/flannel.sh

```diff
@@ -18,10 +18,16 @@
 ETCD_SERVERS=${1:-"http://8.8.8.18:2379"}
 FLANNEL_NET=${2:-"172.16.0.0/16"}
 
+CA_FILE="/srv/kubernetes/etcd/ca.pem"
```

Comment:
Well, it's not a peer certfile for etcd(A peer certfile just like peer-*.pem). Because flannel needs to access the etcd cluster, so...